app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -179,13 +179,37 @@ def reference_to_hash(reference)
 
     def qualifications
       {
-        gcses: qualifications_of_level('gcse').reject(&:missing_qualification?).map { |q| qualification_to_hash(q) },
+        gcses: format_gcses,
         degrees: qualifications_of_level('degree').map { |q| qualification_to_hash(q) },
         other_qualifications: qualifications_of_level('other').map { |q| qualification_to_hash(q) },
         missing_gcses_explanation: missing_gcses_explanation(qualifications_of_level('gcse').select(&:missing_qualification?)),
       }
     end
 
+    def format_gcses
+      gcses = qualifications_of_level('gcse').reject(&:missing_qualification?)
+      parsed_gcses = parse_structured_gcses(gcses)
+      parsed_gcses.map { |q| qualification_to_hash(q) }
+    end
+
+    def parse_structured_gcses(gcses)
+      multiple_english_gcse = gcses.find { |gcse| gcse[:subject] == 'english' && gcse[:structured_grades].present? }
```

Comment:
In this case we would need to check for the science triple award, but other than that it would be nice if it was more generic. I've just pushed an update. Thanks Duncan